### PR TITLE
Stop depending on regions.REGION_CHOICES_SLUG (bug 1058257)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1058257

Use the API instead to fetch the regions list, translated in the current language.

### CODE REVIEW

File: src/templates/includes/chartbody.html [hunk 1]
```diff
@@ -2,16 +2,18 @@
   <form id="rangeform">
     <label>{{ _('From') }} <x-datepicker name="from"></x-datepicker></label>
     <label>{{ _('To') }} <x-datepicker name="to"></x-datepicker></label>
-    {% if regions %}
+    {% if true %}
```
Comment (ngokevin) [inline src/templates/includes/chartbody.html]: yo

File: src/templates/includes/chartbody.html [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
+      {% defer (url=anonApi('regions'), pluck='objects') %}
       <div class="regions">
```
Comment (ngokevin) [inline src/templates/includes/chartbody.html]: indent

File: src/media/js/views/abuse_reports.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ define('views/abuse_reports',
         // L10n: This is the title of a chart representing the number of apps added by payment type.
         var chartTitle = gettext('Abuse Reports');
         var context = {title: chartTitle};
-        if (enableRegions) context.regions = cutils.regions;
+        context.enableRegions = enableRegions;
```
Comment (ngokevin) [inline src/media/js/views/abuse_reports.js]: the number of views are fun huh

File: src/media/js/main.js [hunk 4]
```diff
@@ -5,11 +5,11 @@ init.done(function() {
 require(
     [// Modules actually used in main.
      'core/capabilities', 'core/l10n', 'core/log', 'core/navigation',
-     'core/nunjucks', 'core/user', 'core/z', 'regions', 'user_helpers',
+     'core/nunjucks', 'core/user', 'core/z', 'user_helpers',
      // Modules we require to initialize global stuff.
      'brick', 'd3', 'core/login', 'core/forms', 'helpers_local'],
     function(caps, l10n, log, navigation,
-             nunjucks, user, z, regions, userHelpers) {
+             nunjucks, user, z, userHelpers) {
```
Comment (ngokevin) [inline src/media/js/main.js]: :fries:

**Comment by ngokevin** [comment]
r+wc

File: src/templates/includes/chartbody.html (hunk 1 above)
Comment (diox) [inline src/templates/includes/chartbody.html]: wops :) should be `enableRegions`

**Pull request merged**